app/tickers/eth_gas/queue.ts

```diff
@@ -0,0 +1,90 @@
+import { Job } from 'kue';
+import queue from '../../../config/kue';
+import { logger } from '../../../utils/logger';
+import { oneLine } from 'common-tags';
+import { getJobs } from '../../../utils/jobs_service';
+import { publishActions } from '../../../utils/helpers';
+
+queue.process('eth-tx-worker', function (job, done) {
```

Comment:
Same here, we need a bit specific queue name. Maybe something like `eth-gas-price-worker` etc.

Reply:
Made a commit that changes this value to 'eth-gas-price-worker'